### PR TITLE
chore(postgres-ha): guide update

### PR DESCRIPTION
**Pull request by calinah**
- [x] add instructions about creating a standby and a clone
- [ ] add instructions about restoring from a basebackup
- [ ] mention commands such as `patronictl edit-config` and `patroni reinit`

### CODE REVIEW

File: docs/launchpad/guides/postgresql_ha.md [hunk 1]
```diff
@@ -82,3 +122,214 @@ graph-database:
...
+
+To clone from an S3 bucket you need to define a new PostgreSQL CRD resource with `spec.clone` section defined. Example:
+
+```yaml
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+metadata:
+  name: primary-subgraph-data-clone
+spec:
+  clone:
```
Comment (cjorge-graphops) [inline docs/launchpad/guides/postgresql_ha.md:149]: Maye we want to comment on this section:
- that some of those are optional
- that the default s3_wal_path follows s3://<bucketname>/spilo/<source_db_cluster>/<UID>/wal/<PGVERSION>
- not sure if it applies to clone, but at least in standby's using live instance (cluster, which would be standby_host in that case) is mutually exclusive with using an s3_bucket (so you specify either s3_wal_path or standby_host, but both errors out)

File: docs/launchpad/guides/postgresql_ha.md [hunk 2]
```diff
@@ -82,3 +122,214 @@ graph-database:
...
+
+To start a cluster as a standby from archived WAL files, add the following `standby` section in the Postgres CR definition:
+
+```yaml
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+metadata:
+  name: primary-subgraph-data-clone
+spec:
+  standby:
```
Comment (cjorge-graphops) [inline docs/launchpad/guides/postgresql_ha.md:192]: I think these aren't appropriate for the standby section, should have standby_host and standby_port, and not cluster IIRC

**Review by cjorge-graphops — CHANGES_REQUESTED** [review]
Good work, thank you Ana :bow:

File: docs/launchpad/guides/postgresql_ha.md [hunk 3]
```diff
@@ -6,17 +6,33 @@
...
 
 ## Configuring Postgresql with Zalando's Operator
 
-Launchpad leverages the [Zalando postgres-operator](https://github.com/zalando/postgres-operator) for seamless creation and management of [PostgreSQL](https://www.postgresql.org/) databases within Kubernetes, facilitating highly-available clusters with [Patroni](https://github.com/zalando/patroni). PostgreSQL is a requirement for storing indexer metadata and subgraph data when running an indexer. Configuration is streamlined through Postgres manifests (CRDs), enhancing CI/CD pipeline integration by removing the need for direct Kubernetes API access and promoting infrastructure-as-code practices.
+Launchpad leverages the [Zalando postgres-operator](https://github.com/zalando/postgres-operator) for seamless creation and management of [PostgreSQL](https://www.postgresql.org/) databases within Kubernetes, facilitating highly-available clusters with [Patroni](https://github.com/zalando/patroni). Configuration is streamlined through Postgres manifests (CRDs), enhancing CI/CD pipeline integration by removing the need for direct Kubernetes API access and promoting infrastructure-as-code practices.
```
[the remark this answers is not in the document]
Comment (calinah) [reply docs/launchpad/guides/postgresql_ha.md]: removed

File: docs/launchpad/guides/postgresql_ha.md (hunk 1 above)
Comment (calinah) [reply docs/launchpad/guides/postgresql_ha.md:149]: - added comments in the code for what's optional
- so cluster is required for clones at all times at least according to the docss3://custom/path/to/bucket . Re s3_wal_path not sure if that's correct for clones (following the user docs) but I have updated it on the standby section.
- Re mutual exclusivity it only applies for standbys and I've added a note about the fact it errors out on line 220

File: docs/launchpad/guides/postgresql_ha.md (hunk 2 above)
Comment (calinah) [reply docs/launchpad/guides/postgresql_ha.md:193]: this is for restoring from archived WAL files from s3 lower down ( line 220 ) is for remote live cluster

File: docs/launchpad/guides/postgresql_ha.md (hunk 1 above)
Comment (cjorge-graphops) [reply docs/launchpad/guides/postgresql_ha.md:149]: it's the same bucket be it a clone or a standby so, the same path :p

**Review by cjorge-graphops — APPROVED** [review]

**Review by cjorge-graphops — APPROVED** [review]

**Pull request merged**